app/src/main/java/org/dslul/openboard/inputmethod/latin/spellcheck/AndroidWordLevelSpellCheckerSession.java

```diff
@@ -189,6 +226,7 @@ private static int getCheckabilityInScript(final String text, final int script)
      *  version of it "text" and the capitalized version of it "Text".
      */
     private boolean isInDictForAnyCapitalization(final String text, final int capitalizeType) {
+        updateLocale();
```

Comment:
This one is useless, isInDictForAnyCapitalization() is always called from onGetSuggestionsInternal().